pcdsdevices/tmo_ip1.py

```diff
@@ -0,0 +1,51 @@
+"TMO sample calibration foil to work with spectrometer"
```

Comment:
```suggestion
"""TMO sample calibration foil to work with spectrometer"""
```
Quick nitpick, docstrings at the top of a module use triple quotes.